README.md

```diff
@@ -0,0 +1,154 @@
+# uwv\_dynamic\_model: Implementation of a dynamic model for undewater vehicles
+
+Assumptions:
+
+ - A vehicle, considered a rigid body, is represented by a point in space, been this point the center of gravity (CoG).
+ - The CoG is the origin of the right-hand side body-frame.
+ - There is no representation of multiples objects, neither contact or interaction betwen them.
+ - There is no representation of the environment (floor, surface, water currents, waves, etc)[^TODO].
+ - The vehicle is considered to be surrended by fluid infinitly in all direction.
+ - The gravity field is considered uniform.
+
+## General presentation
+
+The present dynamic model implements the formulation proposed mainly by [Fossen][].
```

Comment:
Maybe we should refer to one of Fossen's book (I'm a classical guy and particular prefer the 94's one)